src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1875,31 +1875,19 @@ public void TestLongProcessIsWorking()
             Assert.True(p.HasExited);
         }
 
-        [PlatformSpecific(TestPlatforms.AnyUnix)]
         [ActiveIssue(37054, TestPlatforms.OSX)]
         [Fact]
         public void GetProcesses_LongProcessName()
         {
-            string commandName = "sleep";
-
-            // sleep program doesn't exist on some flavor
-            // in flavors such as Alpine, sleep is a busybox command which cannot be renamed
-            if (!IsProgramInstalled(commandName) || IsProgramInstalled("busybox"))
-            {
-                return;
-            }
-
-            string longProcessName = "123456789012345678901234567890";
-            string sleepCommandPathFileName = Path.Combine(TestDirectory, longProcessName);
-            File.Copy(GetProgramPath(commandName), sleepCommandPathFileName);
+            string longProcessName = "System.Diagnostics.Process.Tests.SleepAppNative_file_with_quite_a_long_name";
+            string sleepExecutablePath = Path.Combine(TestDirectory, longProcessName);
```

Comment:
@danmosemsft, @safern, I couldn't figure out how to get the custom test native executable into `TestDirectory` for Helix test runs. Is it done by adding it to `Include` or `Exclude` list in `pkg/frameworkPackage.targets` file?

Reply:
Is it listed in the csproj? BTW, I wonder whether we can try to make this new binary more general purpose : reuse it for other tests that might need a native post in future. So for start, not give it a long name, but instead have the test copy it to achieve that.

Reply:
> So for start, not give it a long name, but instead have the test copy it to achieve that.

Added back the `File.Copy` approach. The base name is now the name of the CMake project: `System.Diagnostics.Process.Tests.SleepAppNative` and long name after copy is `System.Diagnostics.Process.Tests.SleepAppNative_file_with_quite_a_long_name` (with `.exe` extension in case of Windows).

Reply:
> Is it listed in the csproj?

I couldn't find any example of existing native binaries produced from `src/Native`, that are listed in project files explicitly. Maybe I am looking at wrong places, is there any existing example?